### DISCUSSION TITLE
m3u file / playlist play random

### DISCUSSION THREAD

**Discussion by jeroenterheerdt** [music-assistant/support#1037] [Q&A] [answered]
I have got a few m3u playlists now, which work great! However, I cannot seem to be able to play them randomly - any other album / artist I can play randomly, but not the playlist. Is this something that could be added in the future

**Comment by OzGav** [comment, marked as answer]
It is not the playlist that is shuffled it is the player queue that the playlist is added to. You need to select the player device, go to the queue and select shuffle on and then add the playlist.

**Comment by jeroenterheerdt** [comment]
I see now, thanks for explaining. Kind of cumbersome, but it works.

**Reply by OzGav** [reply]
It is how all media players work. You just might have more than one with MA/HA so yes you need to enable that on all players you wish to use.

**Comment by jeroenterheerdt** [comment]
Well, not all. Plenty out there that allow you to just play a Playlist randomly, effectively wiping the queue, setting it in random mode and adding the Playlist. Same steps, but one button. Since it's there in Mass for an album / artist, I figured why not for a Playlist.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by OzGav** [reply]
What do you mean when you say it’s there for an album / artist?
